### PR TITLE
Fix goreleaser deprecations

### PR DESCRIPTION
**Pull request by catdevman**
Closes [Issue 378](https://github.com/stateful/runme/issues/378)

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -47,11 +47,11 @@ jobs:
         uses: google-github-actions/setup-gcloud@v0
       - name: Release notes
         run: |
-          go run ./cmd/release-notes/main.go -version "${GITHUB_REF_NAME}" > ${{ runner.temp }}/releasenotes
+          go run ./cmd/release-notes/main.go -version "${GITHUB_REF_NAME}" -owner "${GITHUB_ACTOR}" > ${{ runner.temp }}/releasenotes
```
Comment (catdevman) [inline .github/workflows/release.yml:50]: @sourishkrout I added a new secret... I can revert this but it was basically the only way I could get pass this so it makes the gorelease more flexible for sure.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -60,9 +60,10 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           HOMEBREW_TAP_GITHUB_TOKEN: ${{ secrets.HOMEBREW_TAP_GITHUB_TOKEN }}
           SCOOP_BUCKET_GITHUB_TOKEN: ${{ secrets.SCOOP_BUCKET_GITHUB_TOKEN }}
+          GS_BUCKET: ${{ secrets.GS_BUCKET }}
       - name: Copy to latest
         if: env.pre_release == 'false'
-        run: gsutil -m cp "gs://runme-artifacts-41eac6/${{ env.version }}/*" gs://runme-artifacts-41eac6/latest
+        run: gsutil -m cp "gs://${{ secrets.GS_BUCKET }}/${{ env.version }}/*" gs://${{ secrets.GS_BUCKET }}/latest
```
Comment (catdevman) [inline .github/workflows/release.yml:66]: @sourishkrout same deal here added a new secret to allow me to be able to run the workflow from my own repo

**Comment by catdevman** [comment]
Before I changed the replacements

image 


After removing replacements and using the templating
image

**Comment by sourishkrout** [comment]
Thanks for the contribution @catdevman. This looks great! @duvanmonsa can you please take a stab at a review?

**Review by pastuxso — APPROVED** [review]

**Review by duvanmonsa — APPROVED** [review]
LGTM

**Pull request merged**